core/src/components/datetime/test/disabled/datetime.e2e.ts

```diff
@@ -0,0 +1,156 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not differ across
+ * modes/directions.
+ */
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
+  test.describe(title('datetime: disabled'), () => {
+    test('should not change value when date is clicked', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-datetime value="2022-02-28" disabled></ion-datetime>
+    `,
+        config
+      );
+
+      const ionChange = await page.spyOnEvent('ionChange');
+      await page.waitForSelector('.datetime-ready');
+
+      const febFirstButton = page.locator(`.calendar-day[data-day='1'][data-month='2']`);
+
+      await expect(febFirstButton).not.toHaveClass(/calendar-day-active/);
+
+      await febFirstButton.click({ force: true });
+      await page.waitForChanges();
+
+      await expect(febFirstButton).not.toHaveClass(/calendar-day-active/);
+      await expect(ionChange).not.toHaveReceivedEvent();
+    });
+
+    test('should not navigate months via month-year button', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-datetime value="2022-02-28" disabled></ion-datetime>
+    `,
+        config
+      );
+
+      await page.waitForSelector('.datetime-ready');
+      const calendarMonthYear = page.locator('ion-datetime .calendar-month-year');
+      await expect(calendarMonthYear).toHaveText('February 2022');
+
+      await calendarMonthYear.click({ force: true });
```

Comment:
You can do a `toBeDisabled` assertion here too. Note that you'll need to grab the native button instead since it only works on native control elements:

```js
expect(calendarMonthYear.locator('button')).toBeDisabled();
```

Then you could get rid of the checks below it.